cluster-scope/base/user.openshift.io/groups/data-science/group.yaml

```diff
@@ -2,5 +2,16 @@
 apiVersion: user.openshift.io/v1
 kind: Group
 metadata:
-  name: data-science
-users: []
+    name: data-science
+users:
+    - oindrillac
+    - aakankshaduggal
+    - Shreyanand
+    - MichaelClifford
+    - chauhankaranraj
+    - SankBad
+    - goern
+    - pacospace
+    - durandom
+    - martinpovolny
```

Comment:
I think we can remove `SankBad` and `martinpovolny`

Reply:
@4n4nd hm... what about we do a 1to1 mapping first, so we can be really sure we did not forget anyone and do a clean up later?

I'm worried we might accidentally remove somebody.